src/App.js

```diff
@@ -1,12 +1,56 @@
 import './App.css';
+import React, { Component } from 'react'
 import GitHubButton from './githubButton';
+import ModalButtion from './ModalButtion';
 
-function App() {
-  return (
-    <div className="App">
-      <GitHubButton/>
-    </div>
-  );
+const { Octokit } = require("@octokit/rest");
+const apiKey = `${process.env.REACT_APP_API_KEY}`
+export const octokit = new Octokit({ 
+    auth: apiKey,
+  });
+
+// handleBoolFlagUpdate(boolFlag) {
+//   let newState = this.state;
+//   newState.boolFlag = boolFlag;
+//   console.log(boolFlag)
+//   this.setState(newState)
+// }
+
+// function App() {
+//   return (
+//     <div className="App">
+//       <GitHubButton/>
+//       <ModalButtion/>
+//     </div>
+//   );
+// }
+
+class App extends Component {
```

Comment:
This is fine but generally `App.js` is supposed to be pretty barebones and stateless. I'll probably wrap the stateful information into a container component which will manage everything.